etc/default/default_config.yaml

```diff
@@ -99,23 +99,23 @@ inference_engine:
   name: huggingface
   url: null
   api_key: null
-  model_name: </PATH/TO/INSTALLDIR/INFERENCE_ENGINE>
+  model_name: </PATH/TO/INSTALLDIR/INFERENCE_ENGINE> (e.g. meta-llama/Meta-Llama-3.1-8B-Instruct)
   device: cuda
-  load_in_8bit: true
+  load_in_8bit: false
   max_new_tokens: 150
-  trust_remote_code: true
+  trust_remote_code: false
```

Comment:
Do you recommend setting this to `false` by default? If so let me know so I can update the ansible template as well